docker-compose.yaml

```diff
@@ -282,6 +282,15 @@ services:
       - prom_data:/prometheus
     profiles: ["observability"]
 
+  cadvisor:
```

Comment:
Should this have the `profiles: ["observability"]` tag?

Reply:
The cadvisor container needs to be running anywhere there are docker containers running. Not sure what profile it should have, please let me know